typescript/sdk/src/deploy/core/HyperlaneCoreDeployer.ts

```diff
@@ -68,21 +69,34 @@ export class HyperlaneCoreDeployer<
       for (const remote of remotes) {
         const multisigIsmConfig = this.configMap[remote].multisigIsm;
         const domain = ChainNameToDomainId[remote];
+        let lastTx;
         for (const validator of multisigIsmConfig.validators) {
           const isValidator = await multisigIsm.isEnrolled(domain, validator);
           if (!isValidator) {
             this.logger(
               `Enrolling ${validator} as ${remote} validator on ${chain}`,
             );
-            await multisigIsm.enrollValidator(domain, validator);
+            lastTx = await multisigIsm.enrollValidator(
+              domain,
+              validator,
+              chainConnection.overrides,
+            );
           }
         }
+        if (lastTx) {
+          await chainConnection.handleTx(lastTx);
+        }
         const threshold = await multisigIsm.threshold(domain);
         if (!threshold.eq(multisigIsmConfig.threshold)) {
           this.logger(
             `Setting ${remote} threshold to ${multisigIsmConfig.threshold} on ${chain}`,
           );
-          await multisigIsm.setThreshold(domain, multisigIsmConfig.threshold);
+          const tx = multisigIsm.setThreshold(
+            domain,
+            multisigIsmConfig.threshold,
+            chainConnection.overrides,
+          );
+          await chainConnection.handleTx(tx);
```

Comment:
these are so easy to miss :(

Reply:
Technically they're not *that* necessary if you just want to await a single confirmation it's usually fine